common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1238,7 +1238,7 @@ def submit_problem(self, data, override_time=False):
             # without a stack trace
             else:
                 # Translators: {msg} will be replaced with a problem's error message.
-                msg = _(u"Error: {msg}").format(msg=inst.message)
+                msg = _(u"{msg}").format(msg=inst.message)
```

Comment:
You no longer need to pass "msg" to _, nor use format. The message should already be translated.